### DISCUSSION TITLE
Support importing modules from outside of the root emails/ folder

### DISCUSSION THREAD

**Discussion by callumdmay** [resend/react-email#584] [Ideas]
Hey guys, really love this package, it's so useful especially with the tailwind support. I'm trying to import a file from outside the `emails/` folder into one of my email templates, however when I go to preview it, next.js is throwing an error:

```
Module not found: Can't resolve 'src/util/file.ts'
```
Looks like it might be because of how the next.js server instance is configured to resolve modules:
```
Import trace for requested module:
./emails/ lazy ^\.\/.*$ namespace object
./src/pages/preview/[slug].tsx
```

I can fix this by simply moving file.ts to emails/file.ts but that isn't ideal because file.ts is used a utility file used elsewhere

**Comment by Bugs5382** [comment]
I had to keep my additional folders for helpers, etc. inside the emails folder.

```
emails
- helpers
```
As long as it's a valid react element, it will work.

**Comment by callumdmay** [comment]
This does work, but ideally I would like to avoid copying my helper util files into the email folder
